rasa/core/training/interactive.py

```diff
@@ -1130,9 +1131,22 @@ async def _validate_action(
 
     Returns `True` if the prediction is correct, `False` otherwise."""
 
-    question = questionary.confirm(f"The bot wants to run '{action_name}', correct?")
-
-    is_correct = await _ask_questions(question, conversation_id, endpoint)
+    if action_name == ACTION_UNLIKELY_INTENT_NAME:
+        print(
+            f"The bot wants to run '{action_name}' "
+            f"(https://rasa.com/docs/rasa/default-actions#action_unlikely_intent) "
+            f"to indicate that this user message was unexpected "
+            f"given the context of the conversation. "
+            f"Check out IntentTEDPolicy to learn more. "
+            f"Press any key to continue…"
+        )
+        input()
+        is_correct = True
```

Comment:
I think this part of the GH solution is not implemented:`Users can then choose to confirm / deny the action prediction`, is that something that was decided after the issue submission?

Reply:
It's related to my question above.